spec/models/carousel_list_spec.rb

```diff
@@ -34,6 +34,15 @@ def body(file_name)
   it "has a call_number" do
     expect(subject.call_number).to eq("Z 253 .U6 1963")
   end
+  it "has an isbn" do
+    expect(subject.isbn).to eq(nil)
+  end
+  it "has an issn" do
+    expect(subject.issn).to eq(nil)
+  end
```

Comment:
On these I'd overwrite the part in `@catalog_doc` that should have isbn and issn. So for isbn I'd have a line like

```ruby
@catalog_doc["isbn"] = ["1-5011-8342-7" ]
```

and then expect that value on the next one. 

Same for issn. 

since there is one for oclc you don't have to worry about it.